sailor/assetcentral/system.py

```diff
@@ -243,14 +218,14 @@ def map_component_information(self, selection):
         if len(selection) == 0:
             # build selection dictionary from one of the systems
             intersection = True
-            selection = System._create_selection_dictionary(self[0].components, (self[0].model_id, 1))
+            selection = System._create_selection_dictionary(self[0]._component_tree, (self[0].model_id, 0))
```

Comment:
Zero-based indexing/ordering here --> also change 1 -> 0 in line 98, correct? 
`self._components, equipment_ids, system_ids = System._traverse_components(comps, 1, [], [])`